README.md

```diff
@@ -49,7 +49,7 @@ dotnet ef database update --context OrganisationRegistryContext
 #### To copy events from staging to local db:
 
 ```bash
-bcp OrganisationRegistry.OrganisationRegistry.Events out ./events.bcp -S<server> -U<user> -n -E
+bcp OrganisationRegistry.Events out ./events-staging.bcp -S127.0.0.1,19001 -U <username> -d organisation-registry -n -E
```

Comment:
I'd leave server and port out of the repo. This is subject to local setup, and might be different for different people.